File: Back End/file-storage-service/src/main/java/com/theelite/s3/communication/DeviceService.java
package com.theelite.s3.communication;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface DeviceService {

    @GET("getFamilyAccountForDevice/{id}")
    Call<String> getFamilyAccountForDeviceWithId(@Path("id") String id);
}

File: Front End/Android/app/src/main/java/com/theelite/portal/request/DeviceService.kt
package com.theelite.portal.request

import com.theelite.portal.Objects.Command
import com.theelite.portal.Objects.Device
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.GET
import retrofit2.http.POST
import retrofit2.http.Path

interface DeviceService {

    @GET("device/{email}/{token}")
    fun getDevices(@Path("email") email: String, @Path("token") token : String) : Call<List<Device>>

    @POST("device/command/{email}/{token}")
    fun sendCommand(@Body command: Command, @Path("email") email: String, @Path("token") token : String) : Call<String>
}